### PR TITLE
Redefines urination behaviour

### LINKED ISSUE #14: behavioral control of urination - too species specific?
```
"ID: NBO:0000040
Preferred Name: behavioral control of urination

Synonyms : micturition EXACT []

Definitions     
The regulation of body fluids process by which parasympathetic nerves stimulate 
the bladder wall muscle to contract and expel urine from the body. 
[wikipedia:urination]:

Reference to 'parasympathetic nerves' and 'bladder wall' probably makes this 
too species specific.

```

Original issue reported on code.google.com by `dosu...@gmail.com` on 24 Feb 2013 at 9:17

### PR DESCRIPTION
**Pull request by DitchingIt**
Fixes #14 by removing reference to the bladder and redefining it.

### CODE REVIEW

**Comment by DitchingIt** [comment]
Hi @dosumis @pmidford
Just following up to see if you've had a chance to look at this?
Thanks

File: src/ontology/nbo-edit.owl [hunk 1]
```diff
@@ -1507,17 +1507,17 @@ SubClassOf(obo:NBO_0000039 ObjectSomeValuesFrom(<http://purl.obolibrary.org/obo/
 
 # Class: obo:NBO_0000040 (behavioral control of urination)
 
-AnnotationAssertion(Annotation(oboInOwl:source "wikipedia:urination") obo:IAO_0000115 obo:NBO_0000040 "The regulation of body fluids process by which parasympathetic nerves stimulate the bladder wall muscle to contract and expel urine from the body.")
+AnnotationAssertion(Annotation(oboInOwl:source "wikipedia:urination") obo:IAO_0000115 obo:NBO_0000040 "Excretion behaviour that controls liquid waste expulsion.")
+AnnotationAssertion(dc:contributor obo:NBO_0000040 "https://orcid.org/0000-0002-4277-2151")
```
Comment (matentzn) [inline src/ontology/nbo-edit.owl:1511]:
```suggestion
AnnotationAssertion(dc:contributor obo:NBO_0000040 <https://orcid.org/0000-0002-4277-2151>)
```

**Review by matentzn — COMMENTED** [review]
I made a small comment. Just to position myself:

```
<http://purl.obolibrary.org/obo/nbo#is_about>
```

Is a bad unspecific relationship. But this is not your problem :) PR looks good, made one comment. I will wave it through if they others don't respond.

**Comment by pmidford** [comment]
Definitely an improvement.

**Review by pmidford — APPROVED** [review]
Definitely an improvement, OK to merge.  We'll have to deal with is_about in other places as well I assume.

**Review by matentzn — APPROVED** [review]

**Pull request merged**